{
  "_args": [
    [
      "webpack-fail-plugin@2.0.0",
      "/Users/kkatz/Desktop/pokemon-challenge"
    ]
  ],
  "_development": true,
  "_from": "webpack-fail-plugin@2.0.0",
  "_id": "webpack-fail-plugin@2.0.0",
  "_inBundle": false,
  "_integrity": "sha512-yLeD5BQGtsJM910SXGaos7VbwbY90e/EboyGK1DXaV0+xw08BWDdjALqWcD3UP8yGeAmCBQUnqIE69Y9zwoqcw==",
  "_location": "/webpack-fail-plugin",
  "_phantomChildren": {},
  "_requested": {
    "type": "version",
    "registry": true,
    "raw": "webpack-fail-plugin@2.0.0",
    "name": "webpack-fail-plugin",
    "escapedName": "webpack-fail-plugin",
    "rawSpec": "2.0.0",
    "saveSpec": null,
    "fetchSpec": "2.0.0"
  },
  "_requiredBy": [
    "#DEV:/"
  ],
  "_resolved": "https://registry.npmjs.org/webpack-fail-plugin/-/webpack-fail-plugin-2.0.0.tgz",
  "_spec": "2.0.0",
  "_where": "/Users/kkatz/Desktop/pokemon-challenge",
  "author": {
    "name": "Tiddo Langerak",
    "email": "tiddolangerak@gmail.com"
  },
  "bugs": {
    "url": "https://github.com/TiddoLangerak/webpack-fail-plugin/issues"
  },
  "description": "Webpack plugin that makes the process return an error code on failure",
  "engines": {
    "node": ">=0.11.12"
  },
  "homepage": "https://github.com/TiddoLangerak/webpack-fail-plugin#readme",
  "keywords": [
    "webpack"
  ],
  "license": "MIT",
  "main": "index.js",
  "name": "webpack-fail-plugin",
  "peerDependencies": {
    "webpack": "^1.9.11 || ^2 || ^3"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/TiddoLangerak/webpack-fail-plugin.git"
  },
  "version": "2.0.0"
}
